docs/people-and-groups/changing-session-expiration.md

```diff
@@ -6,9 +6,9 @@ redirect_from:
 
 # Session expiration
 
-By default, Metabase sessions are valid for two weeks after a user last authenticated (e.g. by entering their email
-address/password or via an SSO provider). For example, even if you visit your Metabase instance every day, you'll
-still have to log in again every two weeks.
+{% include plans-blockquote.html feature="Session expiration" %}
+
+By default, Metabase sessions are valid for two weeks after a user last authenticated (e.g. by entering their email address/password or via an SSO provider). For example, even if you visit your Metabase instance every day, you'll still have to log in again every two weeks.
 
 This "session expiration" is configurable via the environment variable `MAX_SESSION_AGE` or as a Java system property:
```

Comment:
We should add `Session inactivity timeout` documentation here.

It can be set both in the UI: Admin > Authentication or env var `MB_SESSION_TIMEOUT`
https://www.metabase.com/docs/latest/configuring-metabase/environment-variables#mb_session_timeout

Pro/EE only